ios-training-bjung/Common/Mapper.swift

```diff
@@ -7,40 +7,37 @@
 
 import Foundation
 
-final class Mapper {
-    static let shared = Mapper()
-    private init() {}
+struct Mapper {
```

Comment:
https://github.com/yumemi-inc/ios-training-bjung/pull/6#discussion_r1436177150

上記のコメントにする対応です。
Mapper を Struct にして、関数や変数を static に変更しました。

Reply:
最近の Swift の主流の中には、今回の `Mapper` のようにインスタンス化の必要がない場面では、`case レス列挙型` を使うことで、インスタンス化できないことを担保する表現方法もあります。

```suggestion
enum Mapper {
```

Reply:
Case 無しの列挙型を使わない場合は、使う人が `Mapper` をインスタンス化する必要があるのか迷わないように、以前に定義していたのと同じ方法 (`private init() {}`) で、イニシャライザーを隠蔽しておくのが良いと思われます。

なお、構造体のイニシャライザー宣言はクラスとは異なり「実装部分でのイニシャライザー定義を省略すると、全てのプロパティーを初期化するための全項目イニシャライザー（今回はプロパティーがないので既定イニシャライザー）が自動実装」されます。

Reply:
@es-kumagai 
コメントありがとうございます。
737b83cbadb699b3cf528f903ece39f8a0b39206
上記のコミットで対応しました。
Case 無しの enum は想像もしてなかった方法ですね・・・